spec/features/expense_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
```

Comment:
- [ ] Please as mentioned by the previous reviewer you should fix the problem with the test, when running tests we got an error about `Bullet.enable`. Kindly fix that 🙏 

<img width="1072" alt="Screen Shot 2023-03-11 at 21 15 14" src="https://user-images.githubusercontent.com/74501523/224504979-966338e6-d131-4b54-820d-c1815e50983c.png">